### PR TITLE
update youtube and website editing instructions

### PR DESCRIPTION
**Pull request by Aariq**
## Content changes/additions

Originally intended on just updating the website editing instructions to remove an outdated step, but realized that the YouTube upload instructions needed changes as well.  Will finalize when I can actually go through these steps

### CODE REVIEW

File: workshops.qmd [hunk 1]
```diff
@@ -58,7 +58,7 @@ From the meeting page, copy the registration link to be included in your worksho
 
 ### Advertise
 
--   Use [this spreadsheet](https://docs.google.com/spreadsheets/d/155tcna1dMWwsZymdKFZPCrnZf3xocw4hPv1JKwOu9MQ/edit#gid=0) to keep track of where you have advertised. 
+-   Use [this spreadsheet](https://docs.google.com/spreadsheets/d/155tcna1dMWwsZymdKFZPCrnZf3xocw4hPv1JKwOu9MQ/edit#gid=0) to keep track of where you have advertised.
```
Comment (Aariq) [inline workshops.qmd:61]: just FYI this is not me being super pedantic—this was the visual markdown editor's change.

File: workshops.qmd [hunk 2]
```diff
@@ -116,20 +118,24 @@ Still a work-in-progress, but should ask questions about:
...
 
 ### Share materials
 
 -   Submit a pull request to [cct-datascience/CALS-workshops](https://github.com/cct-datascience/CALS-workshops) with notes and/or slides
 -   Update workshop info on our [workshops website page](https://datascience.cct.arizona.edu/workshops)
-    -   Under **Category**, add "ALVSCE Workshop Wednesdays" option
+    -   Under **Category**, add "ALVSCE Workshop Wednesdays" option (or other relevant category)
     -   Under **Additional Links**, add slides, notes, etc., with corresponding URLs
-    -   Under **Recording**, add URL of YouTube recording of workshop and **Title** to "View Recording"
-    -   Check box under **Archive**
```
Comment (Aariq) [inline workshops.qmd:131]: Confirmed that archiving is now automated and this checkbox no longer exists

Comment (KristinaRiemer) [reply workshops.qmd:131]: This maybe didn't work for ggplot part 3, so I'm asking Hannah.

File: workshops.qmd [hunk 3]
```diff
@@ -115,21 +117,26 @@ Still a work-in-progress, but should ask questions about:
...
+-   If the video is unedited from the Zoom recording, you can upload the exported .vtt file for closed captions with timings (click "add subtitles", then "upload file", then "with timing")
+-   If possible, include thumbnail from a relevant slide
+-   If you'd like to edit it on YouTube, consider publishing it as "unlisted" initially, but remember to make it public after the edits have processed
 -   Add to the 'Workshops' playlist
 -   Once it's uploaded (may take a while), including the HD version, check that the video is clear enough to see text on screen
--   Advertise video upload via Twitter!
+-   Advertise video upload via Mastodon!
+
+::: callout-tip
+One benefit of editing the video on YouTube studio is that you can upload the .vtt file with a timed transcript along with the original video and then when you trim the video, the subtitles will be trimmed along with it.
```
Comment (KristinaRiemer) [inline workshops.qmd:128]: This is so good to know! I'll definitely do it next time.

**Pull request merged**